charts/novelty/values.yaml

```diff
@@ -111,5 +111,30 @@ thatdot:
   # Cluster-wide log level (ERROR, WARN, INFO, DEBUG)
   logLevel: WARN
 
+webserver:
+  enabled: true
+  address: 0.0.0.0
+  port: 8080
+  useTls: false
+  useMTls:
+    enabled: false
+    healthEndpoints:
+      enabled: false
+      port: 8081
+    # trustStore:
+    #   path: ""
+    #   password: ""
+
+webserverAdvertise:
+  enabled: false
+  address:
+  port:
+
+# mTLS Configuration
```

Comment:
Consider moving this documentation comment directly in the mtls section. That is the typical convention for helm values.yaml files.